### PR TITLE
luci-app-ddns: various fix

### PR DESCRIPTION
**Pull request by Ansuel**
Rework overview page, sync with new style and simplify the code by removing unnecessary check

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

**Comment by pleasantone** [comment]
Who needs to approve/merge this?

File: applications/luci-app-ddns/luasrc/view/ddns/system_status.htm [hunk 1]
```diff
@@ -95,56 +89,62 @@
...
+					'<strong>' + configuration + '</strong>',
+					next_update,
+					lookup,
+					registered_ip,
+					network
+					]);
+					
+			if (rows.length > 1) {
+				var legendtb = document.getElementById('ddns_status_table_legend');
+				legendtb.setAttribute("style", "display:table-row");
```
Comment (jow-) [inline applications/luci-app-ddns/luasrc/view/ddns/system_status.htm]: Why is this needed?

File: applications/luci-app-ddns/luasrc/view/ddns/system_status.htm [hunk 2]
```diff
@@ -95,56 +89,62 @@
...
+			function(x, data) {
+				if (data) { _data2elements(x, data); }
+			}
+		);
+		this.disconnect()
+	});
+	
+	mutationObserver.observe(document.getElementById("wifi_assoclist_table"), {
+		childList: true,
+	});
```
Comment (jow-) [inline applications/luci-app-ddns/luasrc/view/ddns/system_status.htm]: Please remove the `MutationObserver` stuff, the XHR stuff has been fixed in a generic way now.

File: applications/luci-app-ddns/luasrc/view/ddns/system_status.htm (hunk 1 above)
Comment (Ansuel) [reply applications/luci-app-ddns/luasrc/view/ddns/system_status.htm]: I notice that the legend is displayed anyway even if there are no data to show...  it should be hided since is useless...

File: applications/luci-app-ddns/luasrc/view/ddns/system_status.htm (hunk 2 above)
Comment (Ansuel) [reply applications/luci-app-ddns/luasrc/view/ddns/system_status.htm]: so i can keep the xhr get ?

Comment (jow-) [reply applications/luci-app-ddns/luasrc/view/ddns/system_status.htm]: No, just the sole XHR.poll(), no extraneous get etc. required.

**Comment by Ansuel** [comment]
@jow- i think that the legend change should be applied globally and not only here... 
reworked other changes requested

**Comment by jow-** [comment]
I think the table column headers should stay, there is generic placeholder handling in the cbi_update_table() helper. Please remove the special casing from the PR.

**Comment by Ansuel** [comment]
now should be good

**Comment by Ansuel** [comment]
@jow- can you merge this?

**Comment by Ansuel** [comment]
@jow- @hnyman any news on this???

**Comment by Ansuel** [comment]
Can this be merged ?

**Comment by Ansuel** [comment]
@jow- can you review also this?

**Comment by jow-** [comment]
Merged, thanks!
